### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by binary64** [shuckster/eslint-plugin-big-number-rules#3]
Appriciate your plugin
```
TypeError: Cannot convert undefined or null to object
Occurred while linting C:\p\monorepo\apps\src\app\contact\communications-scheduled.service.ts:23
    at valueOf (<anonymous>)
    at C:\p\monorepo\node_modules\eslint-plugin-big-number-rules\dist\index.js:7:3903
    at C:\p\monorepo\node_modules\eslint\lib\linter\safe-emitter.js:45:58
```
This is version 1.7.6

**Comment by shuckster** [comment]
Thanks for the report. 

I think I'll produce a dev version of the plug-in to help with identifying problem areas in the source.

I'll try and address this one without that, but if you can provide a minimal example to reproduce that would be useful.

**Comment by shuckster** [comment]
Hey @binary64 - I think I do need a bit more information for this. I can't figure out how to reproduce the problem.

If you don't want to post the whole `communications-scheduled.service.ts` file, perhaps post what's at line 23 inside it?

**Comment by shuckster** [comment]
@binary64 - Please try `1.9.0`.

I've removed the build process to hopefully improve the error-messages you are getting.

**Comment by shuckster** [comment]
@binary64 - Is this still an issue? I've added an extra guard into [the place](https://github.com/shuckster/eslint-plugin-big-number-rules/blob/master/lib/helpers.js#L157) where I think this is a problem, but not sure if it solves the problem.

**Comment by shuckster** [comment]
Upon investigation this appears to a duplicate of #2.

**Comment by binary64** [comment]
Thanks, this issue appears fixed in the latest version 1.9.1

**Comment by shuckster** [comment]
Nice, thanks for letting me know!
